pom.xml

```diff
@@ -33,6 +33,8 @@
 
     <properties>
         <licenseName>community</licenseName>
+        <maven.compiler.source>1.8</maven.compiler.source>
```

Comment:
I suspect the plugin is already specified in the alfresco-super-pom and it has these variables.
So I think it is not required to configure the plugin again.
If that is not true it would make sense to specify the variables in the plugin configuration instead of hardcoded values.

Reply:
Sure, I just removed it.  Thanks.